components/application-operator/cmd/manager/manager.go

```diff
@@ -159,6 +159,7 @@ func newApplicationReleaseManager(options *options, cfg *rest.Config, helmClient
 		StrictMode:                            options.strictMode,
 		IsBEBEnabled:                          options.isBEBEnabled,
 		PodSecurityPolicyEnabled:              options.podSecurityPolicyEnabled,
+		CentralApplicationConnectivityValidatorEnabled: options.centralApplicationConnectivityValidatorEnabled,
```

Comment:
We need to align the name of this flag as there should be single feature toggle, however, we can postpone it for now.

Reply:
This flag is for override used when application operator update / install help chart for an application gateway.